pkg/config/system_probe.go

```diff
@@ -211,6 +211,7 @@ func InitSystemProbeConfig(cfg Config) {
 	// service monitoring
 	cfg.BindEnvAndSetDefault(join(smNS, "enabled"), false, "DD_SYSTEM_PROBE_SERVICE_MONITORING_ENABLED")
 	cfg.BindEnvAndSetDefault(join(smNS, "process_service_inference", "enabled"), false, "DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_ENABLED")
+	cfg.BindEnvAndSetDefault(join(smNS, "process_service_inference", "use_windows_service_name"), true, "DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_USE_WINDOWS_SERVICE_NAME")
```

Comment:
I enabled this feature by default because `process_service_inference.enabled` is false by default, and `process_context:svchost` isn't very useful. I'm happy to set it to false however if people think we should wait to enable it by default.

Reply:
Addressed offline. It makes sense to keep this feature enabled by default.